RxKeyboard.podspec

```diff
@@ -12,8 +12,8 @@ Pod::Spec.new do |s|
   s.requires_arc     = true
   s.swift_version    = "5.0"
 
-  s.dependency 'RxSwift', '>= 4.4.0'
-  s.dependency 'RxCocoa', '>= 4.4.0'
+  s.dependency 'RxSwift', '>= 5.0.0'
```

Comment:
I think also we should have these as `~> 5.0` because >= 5.0.0 probably also catches 6.x ?

Reply:
Resolved